pallets/duster/Cargo.toml

```diff
@@ -12,36 +12,36 @@ repository = "https://github.com/galacticcouncil/Basilisk-node"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [build-dependencies]
-substrate-wasm-builder = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17" }
+substrate-wasm-builder = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.26" }
 
 [dependencies]
-codec = { default-features = false, features = ["derive"], package = "parity-scale-codec", version = "2.3.1" }
-scale-info = { version = "1.0", default-features = false, features = ["derive"] }
+codec = { default-features = false, features = ["derive"], package = "parity-scale-codec", version = "3.0.0" }
+scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
 serde = { features = ["derive"], optional = true, version = "1.0.136" }
 
-hydradx-traits = { git = "https://github.com/galacticcouncil/warehouse", rev = "f4bdae9b58853bc755f1cbf2b7c794c6375d38f0", default-features = false }
+hydradx-traits = { git = "https://github.com/galacticcouncil/warehouse", branch = "polkadot-v0.9.26", default-features = false }
```

Comment:
i would prefer to use a revision. 

reason is that when this branch is merged, there wont be a need to update this dependency, and we can esily continue to update to another revisions without changing back to "rev =".

Reply:
+1 for revision

Reply:
I disagree with you because with `branch` it's more obvious where it is pointing at, and the branch in warehouse is still open. We are also not aiming at one particular commit. We can easily continue updating this "revision", we will just need to replace `branch` with `rev`. But you clearly outvoted me :) so I'm going to change it

Reply:
@enthusiastmartin do we want to apply this change to all warehouse dependencies?